src/tabris/util-colors.js

```diff
@@ -11,22 +11,31 @@ export function colorStringToArray(str) {
   if (str === 'transparent') {
     return [0, 0, 0, 0];
   }
-  // #xxxxxx
-  if (/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})$/.test(str)) {
+  // #xxxxxx and #xxxxxxxx
+  if (/^#([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})?$/i.test(str)) {
```

Comment:
`a-fA-F` was used on purpose as case insensitive regex are known to be slower.
http://stackoverflow.com/questions/32010/is-regex-case-insensitivity-slower
http://stackoverflow.com/questions/40353587/a-za-z-vs-a-z-i-case-insensitive